README.md

```diff
@@ -363,6 +375,12 @@ type LoaderConstructorOptions = {
   // so some work can be deferred until just before the loader
   // initializes the module.
   importMetaHook?: (fullSpecifier: string, importMeta: Object) => void,
+
+  // Causes the guest  loader to create its own globalThis instead of sharing
+  // its host's.
+  // The constructor copies the own properties over its own globalThis by
+  // assignment.
+  globals: Object,
```

Comment:
Does it resolve #38?

Reply:
No, it expressly does not allow a virtualized global object. The `globalThis` of compartment with a dedicated global object is an ordinary object with a null prototype.

We have in the past received feedback that engines covet the definition of global environment records and that we should not allow that behavior to be virtualized. We would benefit from a fresh confirmation if that’s still the case, cc @codehag, @syg. 

We certainly do not _need_ globalThis to be virtualizable, even for Hardened JavaScript, so we have not asked. However, even if we could pass `globalThis` to a `Loader` (née `Compartment`) constructor, that doesn’t tell the entire story for global environment record and module environment records.

And, we would like to keep the door open if in the future, someone were to champion the use of `Loader` (or `Compartment`) for REPL-style `evaluate`, where each Script can add new `const` and `let` bindings to the global contour (or whatever that’s actually called).

Closer to home, for Hardened JavaScript, it may be useful to introduce a scope that overshadows `globalThis` for both Scripts and Modules that guest code can only access lexically and not discover by enumerating properties of `globalThis`. We used a technique like this to prototype metering, which is similar in spirit to adding an instrumentation/coverage collector in scope.